src/usr/local/www/system_advanced_admin.php

```diff
@@ -238,6 +245,13 @@
 			unset($config['system']['sshdkeyonly']);
 		}
 
+		$sshd_keyonly = isset($config['system']['sshdagentforwarding']);
```

Comment:
Why is this line referring to `$sshd_keyonly`? It looks like it should be `$sshd_agentforwarding`.